plugin.yml

```diff
@@ -5,6 +5,8 @@ requirements:
   - bash
   - docker
   - gpg
+  - curl
+  - shasum
```

Comment:
@joscha `shasum` is only required on Mac, on Linux we need `sha256sum`.

Reply:
Yes, there is no way to express that I think. We can either require both or none. Maybe none is best in that case.